db/fixtures/miq_searches.yml

```diff
@@ -1275,3 +1275,33 @@
           value: analysis_success
     search_type: default
     db: ManageIQ::Providers::InfraManager::Vm
+- attributes:
+    name: State / Running
```

Comment:
a. It's just a name ... as far at I searched, no one use it, so you can write whatever you want here.
b. I will fix it for style :-)

@moolitayer fixing now 👍